cmd/keploy-cli/cmd/root.go

```diff
@@ -0,0 +1,23 @@
+package cmd
```

Comment:
why not just add this file to the server package?

Reply:
Yeah, right. Sorry my bad.